Formula/gnu-prolog.rb

```diff
@@ -1,12 +1,8 @@
 class GnuProlog < Formula
   desc "Prolog compiler with constraint solving"
   homepage "http://www.gprolog.org/"
-  # Normal download page is from the http://www.gprolog.org/, however in October 2020
-  # a slightly updated "1.4.5" version was posted there which broke the sha256 sum
-  # In the next release we can go back to using this as our official source, but
-  # for now download from GNU which still has the original 1.4.5 available:
-  url "https://ftp.gnu.org/gnu/gprolog/gprolog-1.4.5.tar.gz"
-  sha256 "bfdcf00e051e0628b4f9af9d6638d4fde6ad793401e58a5619d1cc6105618c7c"
+  url "https://ftp.gnu.org/gnu/gprolog/gprolog-1.5.0.tar.gz"
```

Comment:
Seems like the stable URL from the homepage is available too – http://www.gprolog.org/gprolog-1.5.0.tar.gz, any reason we're not using that? Asking because of the comment, which states that we should go back to using the one from the homepage.

Reply:
Sure. Updated.